components/energylevel.css

```diff
@@ -0,0 +1,147 @@
+.energylevel {
+  width: 315px;
+  height: 208px;
+  display: grid;
+  grid-template-columns: 32px repeat(5, minmax(32px, 1fr));
+  grid-template-rows: repeat(6, minmax(0, auto));
+  grid-template-areas:
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    '.           time time time time time';
+}
+
+.energylevel h3 {
+  line-height: 1.5;
+  margin: 0;
+  padding: 0;
+}
+
+.energylevel__background {
+  grid-area: bars;
+  margin-top: 16px;
+  background: repeating-linear-gradient(
+    var(--color-gray) 0 1px,
+    var(--color-white) 1px 41px
+  );
+  z-index: -1;
+}
+
+.energylevel__percentage {
+  grid-area: percentage;
+  height: 208px;
+  width: 32px;
+  display: flex;
+  flex-direction: column;
+  align-items: space-around;
+  justify-content: flex-start;
+}
+
+.energylevel__time {
+  grid-area: time;
+  display: flex;
+  justify-content: space-around;
+}
+
+.energylevel__percentage > li {
+  padding-bottom: 24px;
+}
+
+.energylevel__percentage:last-child {
+  padding-bottom: -2px;
+}
+
+.energylevel__bars {
+  grid-area: bars;
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-end;
+}
+
+.energylevel__bars-double-1 {
+  display: flex;
+  align-items: flex-end;
+  margin-bottom: 9px;
+}
+.energylevel__bars-orange-1 {
+  width: 12px;
+  height: 175px;
+  background: var(--color-darkorange);
+  border-radius: 50px 50px 0 0;
+}
+.energylevel__bars-blue-1 {
+  width: 12px;
+  height: 154px;
+  background: var(--color-darkblue);
+  border-radius: 50px 50px 0 0;
+}
```

Comment:
Sich wiederholende Anweisungen wie width, background und border-radius könnte man besser in einer allgemeinen Klasse, z.B. "energylevel__bar" zusammenfassen, um Code zu reduzieren.